.gitlab/deception.gitlab-ci.yml

```diff
@@ -0,0 +1,35 @@
+
+cleanup:
+ stage: .pre
+ extends: .SVC-Account-Cleanup
+
+hykkt-test-deception:
+  variables:
+    SLURM_Q: "a100_shared"
+    MY_CLUSTER: "deception"
+  extends:
+    - .cluster_test 
+
+pending:
+  variables:
+    MACHINE: "deception"
+  extends:
+    - .report-pending
+  stage: .pre
+
+success:
+  variables:
+    MY_CLUSTER: "deception"
+  extends:
+    - .report-status
+  stage: .post
+       
+failure:
+  stage: .post
+  variables:
+    MY_CLUSTER: "deception"
+  extends:
+    - .report-status
+  rules:
+    - when: on_failure
+  
```

Comment:
```suggestion
.deception:
  variables:
    MY_CLUSTER="Deception"
    SLURM_Q="a100_shared,a100_80_shared,dl_shared,dlv_shared,dlt_shared"
    
hykkt-test-deception:
  extends:
    - .deception
    - .cluster_test 

pending:
  extends:
    - .deception
    - .report-pending

success:
  extends:
    - .deception
    - .report-status
       
failure:
  extends:
    - .deception
    - .report-status
  rules:
    - when: on_failure
  
```

Reply:
Maybe there is already an issue, but we could add a list of SLURM partitions if we wanted.